node/pkg/driver/node_test.go

```diff
@@ -1338,6 +1459,16 @@ func assertError(t *testing.T, err error, expectedErrorCode codes.Code) {
 	}
 }
 
+func assertExpectedStats(t *testing.T, expResp *csi.NodeGetVolumeStatsResponse, req *csi.NodeGetVolumeStatsRequest, node driver.NodeService) {
```

Comment:
I would put this next to the `NodeGetVolumeStats` tests, since it's not relevant to other tests

Reply:
done